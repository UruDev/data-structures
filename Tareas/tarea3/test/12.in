# altura_binario, cantidad_binario,kesimo_en_binario
insertar_en_binario (12,m)
insertar_en_binario (8,g)
altura_binario
cantidad_binario
kesimo_en_binario 1
kesimo_en_binario 2
kesimo_en_binario 3
insertar_en_binario (9,j)
insertar_en_binario (2,c)
insertar_en_binario (1,a)
altura_binario
cantidad_binario
kesimo_en_binario 1
kesimo_en_binario 2
kesimo_en_binario 4
insertar_en_binario (4,e)
insertar_en_binario (5,h)
insertar_en_binario (7,k)
insertar_en_binario (14,p)
insertar_en_binario (18,s)
insertar_en_binario (1,r)
insertar_en_binario (20,z)
insertar_en_binario (2,n)
altura_binario
cantidad_binario
kesimo_en_binario 1
kesimo_en_binario 3
kesimo_en_binario 5
kesimo_en_binario 8
kesimo_en_binario 9
kesimo_en_binario 11
kesimo_en_binario 13
Fin
