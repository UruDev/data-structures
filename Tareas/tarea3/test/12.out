1>#  altura_binario, cantidad_binario,kesimo_en_binario.
2>Insertado.
3>Insertado.
4>2.
5>2.
6>(8,g).
7>(12,m).
8>k fuera de rango.
9>Insertado.
10>Insertado.
11>Insertado.
12>4.
13>5.
14>(1,a).
15>(2,c).
16>(9,j).
17>Insertado.
18>Insertado.
19>Insertado.
20>Insertado.
21>Insertado.
22>Insertado.
23>Insertado.
24>Insertado.
25>4.
26>13.
27>(1,a).
28>(4,e).
29>(5,h).
30>(12,m).
31>(2,n).
32>(1,r).
33>(20,z).
34>Fin.
